FoodBoard/Features/Authorization/AuthorizationController.cs

```diff
@@ -0,0 +1,30 @@
+﻿using FoodBoard.Features.Authorization.Representations;
+using Microsoft.AspNetCore.Mvc;
+
+namespace FoodBoard.Features.Authorization
+{
+    [Route("api/[controller]")]
+    public class AuthorizationController : ControllerBase
+    {
+        private readonly IAuthorizationService _authorizationService;
+        private readonly ITokenGenerator _tokenGenerator;
+
+        public AuthorizationController(IAuthorizationService authorizationService, ITokenGenerator tokenGenerator)
+        {
+            _authorizationService = authorizationService;
```

Comment:
nur als frage für mich. Warum der `_` vor manchen variablen?

Reply:
C# Naming convention für mehr Infos 

https://www.c-sharpcorner.com/UploadFile/8a67c0/C-Sharp-coding-standards-and-naming-conventions/